keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -780,6 +783,67 @@ in terms of retriability and evolving Pod condition types
 to do not add any pod condition in this case. It should be re-considered in the
 future if there is a good motivating use-case.
 
+##### Admission failures
+
+In some scenarios a pod admission failure could be restarted and is likely to
+succeed on another node (for example a pod scheduled to a node with resource
+pressure, see: [Pod admission error](#pod-admission-error)). However, in some
+situations isn't clear as the failure can be caused by incompatible pod and node
```

Comment:
How about something like: 

```
In some scenarios a pod admission failure could result in a successful pod restart on another 
node (for example a pod scheduled to a node with resource pressure, see: 
[Pod admission error](#pod-admission-error)). However, in other situations it won't be as clear, 
since the failure can be caused by incompatible pod and node configurations. Node configurations 
are often the same within a cluster, so it is likely that the pod would fail if restarted on any 
other node in the cluster. In that case, adding `DisruptionTarget` condition could cause 
a never-ending loop of retries, if the pod failure policy was configured to ignore such failures. 
Given the above, we decide not to add any pod condition for such failures. If there is a sufficient
motivating use-case, a dedicated pod condition might be introduced to annotate some of the 
admission failure scenarios.
```